components/progressbar/metadata/meter.yml

```diff
@@ -1,34 +1,71 @@
 name: Meter
-description: Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+description: |
+  Meter is implemented using the [Progress bar](progressbar.html) component. Refer to the [Progress bar API docs](progressbar.html) for API details.
+  Meter should be only used in size `S` and `L` and could use the meter t-shirt sized classnames. [The Meter table of options](https://spectrum.adobe.com/page/meter/#Table-of-options) provides further details on which variants are available.
+
+  | Previous size classname         | New size classname                |
+  | ------------------------------- | --------------------------------- |
+  | `.spectrum-ProgressBar--sizeS`  | `.spectrum-Meter--sizeS`    |
+  | `.spectrum-ProgressBar--sizeL`  | `.spectrum-Meter--sizeL`    |
 SpectrumSiteSlug: https://spectrum.adobe.com/page/meter/
+
 examples:
+  - id: meter-sizes
+    name: Sizing
+    markup: |
+      <div style="margin: 20px 0;">
+        <div class="spectrum-ProgressBar spectrum-Meter--sizeS is-positive" value="50" role="progressbar" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100" style="width: 200px;">
+          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeS spectrum-ProgressBar-label">Storage Space</div>
+          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeS spectrum-ProgressBar-percentage">50%</div>
+          <div class="spectrum-ProgressBar-track">
+            <div class="spectrum-ProgressBar-fill" style="width: 50%;"></div>
+          </div>
+        </div>
+      </div>
+      <div style="margin: 20px 0; width: 200px;">
+        <div class="spectrum-ProgressBar spectrum-Meter--sizeSL is-notice" value="80" role="progressbar" aria-valuenow="80" aria-valuemin="0" aria-valuemax="100" style="width: 200px;">
+          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeL spectrum-ProgressBar-label">Storage Space</div>
+          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeL spectrum-ProgressBar-percentage">80%</div>
+          <div class="spectrum-ProgressBar-track">
+            <div class="spectrum-ProgressBar-fill" style="width: 80%;"></div>
+          </div>
+        </div>
+      
   - id: meter-small
-    name: Standard
+    name: Variant options
     markup: |
       <div style="margin: 20px 0;">
-        <div class="spectrum-ProgressBar spectrum-ProgressBar--sizeM is-positive" value="50" role="progressbar" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100">
-          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-ProgressBar-label">Storage Space</div>
-          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-ProgressBar-percentage">50%</div>
+        <div class="spectrum-ProgressBar spectrum-Meter--sizeL is-positive" value="50" role="progressbar" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100">
```

Comment:
Can we also add more context to these examples similar to what I suggested for the t-shirt sizes above?

I think each of these options could be an example with headings such as "Default", "Positive", "Notice", and "Negative" similar to [Status Light's semantic colors](https://opensource.adobe.com/spectrum-css/statuslight.html#positive).

Reply:
Sure, we can place them in individual headers + containers.